apis/python/src/tiledbsoma/soma_collection.py

```diff
@@ -270,7 +270,8 @@ def _determine_default_relative(self, uri: str) -> Optional[bool]:
         if self._tiledb_platform_config.member_uris_are_relative is not None:
             return self._tiledb_platform_config.member_uris_are_relative
         if uri.startswith("tiledb://"):
-            return True
+            # TileDB-Cloud does not use relative URIs, ever.
+            return False
         return None
```

Comment:
why not `return True` here? & i think this should return `bool` not `Optional[bool]`

Reply:
It must be able to also return a None, which is not type bool (see line 275).